sdcm/utils/aws_builder.py

```diff
@@ -26,12 +29,81 @@
 
 LOGGER = logging.getLogger(__name__)
 
+ASG_CONFIG_FORMAT = """
+import com.amazonaws.services.ec2.model.InstanceType
+import hudson.plugins.sshslaves.SSHConnector
+import hudson.plugins.sshslaves.verifiers.NonVerifyingKeyVerificationStrategy
+import hudson.model.*
+import com.amazon.jenkins.ec2fleet.EC2FleetCloud
+import jenkins.model.Jenkins
+
+// just modify this config other code just logic
+config = [
+    name: "{name}",
+    region: "{region_name}",
+    fleetId: "{asg_id}",
+    idleMinutes: 10,
```

Comment:
Is it the idle time before scale down?
Maybe we should consider larger time?

Reply:
why ? it would be killed and a new one would be created.
what do we loose if something like this happens ? we wait 1-2 minutes for a new one to spin up.

Reply:
Yes, I guess.
Hard to tell without seeing it working.